types/echarts/index.d.ts

```diff
@@ -1187,9 +1187,9 @@ declare namespace echarts {
                  * Moreover, it can be set as negative number, like `-3`.
                  *
                  * @default null
-                 * @see https://ecomfe.github.io/echarts-doc/public/en/option.html#yAxis.max
+                 * @see https://echarts.apache.org/zh/option.html#yAxis.max
```

Comment:
I recommend that the the link should point to the English version: e.g. `/en/` instead of `/zh/`

Reply:
Same as before.